python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -339,6 +340,29 @@ async def setup_laser(
         await self.change_laser_optical_configuration(optical_configuration)
         await self.change_laser_wavelength(wavelength, use_projector)
 
+    async def get_projector_setup(self) -> tuple:
+        """Get configuration of flatfield projector
+
+        Returns
+        -------
+            list : configuration details
+        """
+
+        select_location = await self.linearstage_projector_select.position
+        led_location = await self.linearstage_led_select.position
+        led_focus = await self.linearstage_led_focus.position
+        laser_focus = await self.linearstage_laser_focus.position
+        led_state = await self.rem.ledprojector.evt_ledstate.aget()
```

Comment:
You might want to add a timeout here or this might lock forever.

```
led_state = await self.rem.ledprojector.evt_ledstate.aget(timeout=self.fast_timeout)
```